### DISCUSSION TITLE
v0.12

### DISCUSSION THREAD

**Discussion by jianyuan** [jianyuan/terraform-provider-sentry#361] [Announcements]
With Sentry’s support, we’ve made several enhancements to this terraform provider!

You can now do the following with the provider:

1. [Add members to a team](https://registry.terraform.io/providers/jianyuan/sentry/latest/docs/resources/team_member).
2. Add/delete/update org and team level roles (Use [`sentry_team_member`](https://registry.terraform.io/providers/jianyuan/sentry/latest/docs/resources/team_member) for team level roles like Team Admin and [`sentry_organization_member`](https://registry.terraform.io/providers/jianyuan/sentry/latest/docs/resources/organization_member) for org roles like Manager/Billing etc).
3. [Create projects without the default alert rule](https://registry.terraform.io/providers/jianyuan/sentry/latest/docs/resources/project#default_rules).
4. [Add rate limits to project DSN Keys](https://registry.terraform.io/providers/jianyuan/sentry/latest/docs/resources/project#default_key).
5. [Add inbound filters to projects](https://registry.terraform.io/providers/jianyuan/sentry/latest/docs/resources/project_inbound_data_filter).
7. [Add Issue](https://registry.terraform.io/providers/jianyuan/sentry/latest/docs/resources/issue_alert) and [metric alerts](https://registry.terraform.io/providers/jianyuan/sentry/latest/docs/resources/metric_alert) to Sentry - Including sending alerts to Slack/Pagerduty etc.
8. [Enable Spike Protection for your projects](https://registry.terraform.io/providers/jianyuan/sentry/latest/docs/resources/project_spike_protection) and [add notifications for spike protection](https://registry.terraform.io/providers/jianyuan/sentry/latest/docs/resources/notification_action).
9. [Add/update debug symbol server sources for your projects](https://registry.terraform.io/providers/jianyuan/sentry/latest/docs/resources/project_symbol_source).

Please see our docs for the latest version of the provider for more details: https://registry.terraform.io/providers/jianyuan/sentry/latest/docs

**Comment by landenai** [comment]
Hi @jianyuan I dont see the options for adding rate limits to project DSN Keys in the docs. Am I missing something?
